### DISCUSSION TITLE
please help me how I can fix this problem "OpenVPN process exited"

### DISCUSSION THREAD

**Discussion by a4medsoft** [schwabe/ics-openvpn#1382] [General]
please help me how I can fix this problem "OpenVPN process exited"

# General information
1. Android Version 10 or any
2. Android Vendor/Custom ROM
3. Device any devices
4. Version of the app (version number/play store version/self-built)
 
#  how i can fix this problem "OpenVPN process exited"

**Comment by schwabe** [comment]
Please provide a log file
